testGameSearch/GameViewModel.swift

```diff
@@ -12,27 +12,21 @@ import Combine
 class GameViewModel: ObservableObject {
     
     private var gameService: GameService
-    
-    @Published var game = Game()
-    
-    //getting a Missing argument for paramter 'from' in call error here
-    
+    @Published var game: Game?
     
     init() {
         self.gameService = GameService()
         
     }
     
-    var gameName: String = ""
+    @Published var gameName: String = ""
```

Comment:
Can't remember if this is _strictly_ necessary but if your Textfield is using this as the binding for it's string then I believe you want it to be a publisher

Reply:
I deleted the @published and it didn't affect the output just to see but if it's being used I'll leave it in. Publishers are magic.